cli/tests/00-base.result

```diff
@@ -1,9 +1,10 @@
-b	2	false
-a	1	true
-0	99999	100000
+b	2	false	"{""k"":""v""}"
```

Comment:
The result seems weird.

Reply:
this is because the `QuoteStyle` of tsv is `Necessary`, string will add extra comma.